calendar-bundle/src/Resources/contao/classes/Calendar.php

```diff
@@ -231,13 +231,17 @@ protected function generateFiles($arrFeed)
 					$GLOBALS['objPage'] = $this->getPageWithDetails(CalendarModel::findByPk($event['pid'])->jumpTo);
 
 					$objItem = new FeedItem();
-
 					$objItem->title = $event['title'];
 					$objItem->link = $event['link'];
 					$objItem->published = $event['tstamp'];
 					$objItem->begin = $event['startTime'];
 					$objItem->end = $event['endTime'];
 
+					if ($event['isRepeated'] ?? null)
+					{
+						$objItem->guid = $event['link'] . '#' . date('Y-m-d', $event['startTime']);
+					}
```

Comment:
imho it would be fine to always append `#Y-m-d`. Then you would not need a new parameter.

Reply:
No, the first event should have the real URL IMHO.

Reply:
I also think we sould only append the hash for the duplicates, not for the first one. This way we would only get `isPermaLink="false"` for the duplicate URLs but not for the first one.